src/styles/InstructionScreen.css

```diff
@@ -0,0 +1,25 @@
+*{
```

Comment:
I'd recommend placing the CSS file for the instructions screen next to the instructions screen component. This way files that are strongly related are kept close to each other